packages/flame/lib/src/assets/images.dart

```diff
@@ -137,11 +157,11 @@ class Images {
     return completer.future;
   }
 
-  Future<Image> fromBase64(String fileName, String base64) async {
-    if (!_loadedFiles.containsKey(fileName)) {
-      _loadedFiles[fileName] = _ImageAssetLoader(_fetchFromBase64(base64));
+  Future<Image> fromBase64(String key, String base64) async {
```

Comment:
Should be possible to remove the `async` here once the other one is removed

Reply:
done